presto-main/src/main/java/com/facebook/presto/server/CoordinatorModule.java

```diff
@@ -182,6 +183,9 @@ protected void setup(Binder binder)
         // planner
         binder.bind(PlanFragmenter.class).in(Scopes.SINGLETON);
 
+        // optimizers
```

Comment:
is this comment needed?

Reply:
Good point, I'll make this part of "planner".